x-pack/plugins/cloud_security_posture/public/pages/configurations/latest_findings_by_resource/findings_by_resource_container.tsx

```diff
@@ -36,7 +37,8 @@ const getDefaultQuery = ({
   query,
   filters,
   pageIndex: 0,
-  sortDirection: 'asc',
+  sortDirection: 'desc',
```

Comment:
1. Does this change effect the UX of the product? lower compliance scores should be displayed first by default unless the user re-sorts. can you please attach a video here and also for future PRs? it would be very helpful
2. Do we still use `sortDirection` after your changes? it looks like the new `sort` field should be sufficient isn't it? maybe we can remove `sortDirection` completely?

Reply:
@animehart page should start with ascending order - with the idea that the lower score is more relevant to see first when looking at mis configurations